third_party/purge/helpers/varnish_helper.php

```diff
@@ -30,12 +32,20 @@ function send_purge_request($site_url = NULL, $site_port = NULL)
 			$port = 80;
 		}
 		
+		echo $purge_url; 
+		
 		$ch = curl_init();
+		
 		curl_setopt($ch, CURLOPT_URL, $purge_url);
 		curl_setopt($ch, CURLOPT_PORT , (int)$port);
-		curl_setopt($ch, CURLOPT_CUSTOMREQUEST,'EE_PURGE');
+		curl_setopt($ch, CURLOPT_HTTPHEADER, array( 'Host: '.$_SERVER['SERVER_NAME'] ) );
+		curl_setopt($ch, CURLOPT_CUSTOMREQUEST,'PURGE');
 		curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-		curl_exec($ch);
+		curl_setopt($ch, CURLOPT_CONNECTTIMEOUT ,1); 
+		curl_setopt($ch, CURLOPT_TIMEOUT, 2);
```

Comment:
Have increased. just wanted something in case app server can't reach varnish (e.g. firewall problem) didn't want it to take 30+ seconds to save page.